src/execution.jl

```diff
@@ -1,3 +1,20 @@
+
+# CUDArt defines CudaArray
```

Comment:
Idem. If adding to CUDArt, you don't need those `isdefined` checks.

Also, we don't tend to use parentheses around the if conditions.